client/src/components/pages/__tests__/ChannelCreate.test.tsx

```diff
@@ -26,6 +43,40 @@ describe('Rendering', () => {
     expect(json).toBeTruthy();
     expect(json).toMatchSnapshot();
   });
+
+  it('renders [UserListItem]', async () => {
+    const user: AuthProviderMeQueryResponse['me'] | null = {
```

Comment:
```suggestion
    const user: AuthProviderMeQueryResponse['me'] = {
```
I think it is safe to remove `null` here.